src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -455,7 +455,7 @@ protected function doRequestInProcess($request)
             foreach ($deprecations ? unserialize($deprecations) : [] as $deprecation) {
                 if ($deprecation[0]) {
                     // unsilenced on purpose
-                    trigger_error($deprecation[1], \E_USER_DEPRECATED);
+                    @trigger_error($deprecation[1], \E_USER_DEPRECATED);
```

Comment:
```suggestion
                    trigger_error($deprecation[1], \E_USER_DEPRECATED);
```

See line above :smile: